CHANGELOG.md

```diff
@@ -3,7 +3,95 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.0] nfcore/rnavar - 2022/06/20
+## [1.2.0](https://github.com/nf-core/rnavar/releases/tag/1.2.0) - Mighty Milano
```

Comment:
Don't forget to add date ;)

Reply:
Thanks for the reminder, we'll be sure to do that once we release :)

Reply:
I don't like much having a data actually

Reply:
Either way is good to me :)